cmd/end2endtest/helpers.go

```diff
@@ -414,18 +414,17 @@ func (t *e2eElection) checkElectionPrice(ed *vapi.ElectionDescription) error {
 func (t *e2eElection) overwriteVote(choices []int, indexAcct int, waitType string) error {
 	for i := 0; i < len(choices); i++ {
 		// assign the choices wanted for each overwrite vote
-		errs := t.sendVotes([]*apiclient.VoteData{{
+		if _, err := t.api.Vote(&apiclient.VoteData{
```

Comment:
do note that t.api.Vote doesn't handle the "mempool is full" situation, so the overwritetest will fail in that case.

Reply:
Yeah, agree. I think it's worthy.